pkg/root/types.go

```diff
@@ -30,7 +30,7 @@ func (t Target) ToGlob() string {
 	if t.KernelVersion == "" {
 		t.KernelVersion = "*"
 	}
-	return fmt.Sprintf("%s_%s_%s.yaml", t.Distro, t.KernelRelease, t.KernelVersion)
+	return fmt.Sprintf("%s_%s_%s.*", t.Distro, t.KernelRelease, t.KernelVersion)
```

Comment:
Note: make sure that no file other than `.yaml` or `.{ko,o}` are present inside inner folders ;)